nessie/sql_templates/create_edl_schema.template.sql

```diff
@@ -103,17 +103,14 @@ AS (
         s.action_dt AS action_date
     FROM {redshift_schema_edl_external}.student_academic_standing_data s
     JOIN (
-      SELECT student_id, semester_year_term_cd, MAX(action_dt) AS action_dt, 
-          MAX(academic_standing_effective_dt_seq) AS seq_nr, MAX(academic_standing_effective_dt) AS eff_dt
+      SELECT student_id, semester_year_term_cd, 
+      MAX(academic_standing_effective_dt || academic_standing_effective_dt_seq) AS eff_dt_seq
       FROM {redshift_schema_edl_external}.student_academic_standing_data
-      JOIN {redshift_schema_calnet}.advisees a ON a.sid = student_id
```

Comment:
I don't think that this join to filter down to the advisee schema is necessary any longer. Academic standing for all!